ENGAGEHF/Managers/NotificationManager/NotificationRegistrationSchema.swift

```diff
@@ -0,0 +1,51 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import class UIKit.UIDevice
+
+
+enum MobilePlatforms: String {
```

Comment:
I think this is not used anywhere?

Reply:
I user its raw value in the `NotificationRegistrationSchema` initializer. I can just user a string literal instead, but figured it would be nice to define this to avoid typos.